Library/Services/Implementation/NfieldBlacklistService.cs

```diff
@@ -29,6 +29,15 @@ namespace Nfield.Services.Implementation
     /// </summary>
     internal class NfieldBlacklistService : INfieldBlacklistService, INfieldConnectionClientObject
     {
+        public Task<string> GetAsync()
+        {
+            var uri = BlacklistUrl();
+
+            return Client.GetAsync(uri)
+                .ContinueWith(task => task.Result.Content.ReadAsStringAsync().Result)
```

Comment:
Isn't this potentially dangerous (using `Result` on an async method). This should be awaited I think

Reply:
Why is this method not simply `async`?
``` c#
var result = await Client.GetAsync(uri);
var blacklist = await result.Content.ReadAsStringAsync();

return blacklist;
```
Or am I missing some vital bit of info?

Reply:
We also copy-pasted some old code in our PR #352, but then changed it to use `await`.